Dieser "Hack"-Patch ermoeglicht es einem Prozess, sich im Speicher zu
verewigen...

Notwendig, wenn der ISDN-Server auch noch was anderes zu tun hat.

diff -rub --unidir /pub/src/linux/kernel/linux/kernel/sys.c ./kernel/sys.c
--- /pub/src/linux/kernel/linux/kernel/sys.c	Sat Mar  4 23:16:36 1995
+++ ./kernel/sys.c	Wed Mar  8 05:38:27 1995
@@ -146,6 +146,10 @@
 {
 	if (!suser())
 		return -EPERM;
+    if(magic == 0xfee1dead && magic_too == 0x17392634) {
+        current->mm->swappable = flag ? 0 : 1;
+        return 0;
+    }
 	if (magic != 0xfee1dead || magic_too != 672274793)
 		return -EINVAL;
 	if (flag == 0x01234567)
